src/services/groupService.js

```diff
@@ -0,0 +1,58 @@
+import Group from '../models/group';
+import User from '../models/user';
+import userGroups from '../models/userGroup';
+
+const { Sequelize } = require('sequelize');
+const sequelize = new Sequelize('postgres://jypfzbwx:aYUoFzUvmkdbT98u4D0X1RPXkAlxBBQt@lallah.db.elephantsql.com:5432/jypfzbwx');
+
+class GroupService {
+    getAllGroups() {
+        return Group.findAll();
+    }
+
+    getGroupById(id) {
+        return Group.findByPk(id);
+    }
+
+    createGroup(groupDTO) {
+        return Group.create(groupDTO);
+    }
+
+    updateGroup(id, groupDTO) {
+        return Group.update(groupDTO, { where: { id } });
+    }
+
+    deleteGroup(id) {
+        return Group.destroy({ where: { id } });
+    }
+
+    async addUsersToGroup(groupId, usersId) {
+        console.log(usersId);
+        if (!groupId || !usersId || !Array.isArray(usersId)) {
+            const error = new Error('Wrong addUsersToGroup parameters.');
+            error.status = 404;
```

Comment:
Adding http status code is not a service responsibility. You can simply throw error from here